src/lib/button/_button-base.scss

```diff
@@ -66,7 +66,9 @@ $mat-mini-fab-padding: 8px !default;
 %mat-raised-button {
   @extend %mat-button-base;
 
-  @include mat-elevation(2);
+  &:not([class*='#{$_mat-elevation-prefix}']) {
```

Comment:
Would it make sense to make a mixin for this? Something like...
```scss
@mixin mat-overridable-elevation($z) {
  &:not([class*='#{$_mat-elevation-prefix}']) {
    @include mat-elevation(2);
  }
}
```

Reply:
Good idea, that selector was pretty funky. Added.